Configuration/TCA/tx_tracking_recordview.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+return [
+    'ctrl' => [
+        'label' => 'record',
+        'label_alt' => 'crdate',
+        'label_alt_force' => true,
+        'default_sortby' => 'crdate DESC',
+        'tstamp' => 'tstamp',
+        'crdate' => 'crdate',
+        'cruser_id' => 'cruser_id',
+        'languageField' => 'sys_language_uid',
+        'title' => 'LLL:EXT:tracking/Resources/Private/Language/locallang_tca.xlf:table.recordview',
+        'searchFields' => 'uid, url',
+        'iconfile' => 'EXT:core/Resources/Public/Icons/T3Icons/apps/apps-pagetree-page-default.svg',
+    ],
+    'types' => [
+        '0' => [
+            'showitem' => 'sys_language_uid, pid, record, url, user_agent, operating_system, crdate',
+        ],
+    ],
+    'columns' => [
+        'pid' => [
+            'label' => 'LLL:EXT:tracking/Resources/Private/Language/locallang_tca.xlf:table.recordview.pid',
+            'config' => [
+                // TYPO3 v10 does no longer allow to resolve PID relations, e.g. via select or group
+                // This will break internal PID handling.
+                'type' => 'input',
+                'readOnly' => true,
+            ],
+        ],
+        'crdate' => [
+            'label' => 'LLL:EXT:tracking/Resources/Private/Language/locallang_tca.xlf:table.recordview.crdate',
+            'config' => [
+                'type' => 'input',
+                'eval' => 'datetime',
+            ],
+        ],
+        'sys_language_uid' => [
+            'label' => 'LLL:EXT:tracking/Resources/Private/Language/locallang_tca.xlf:table.recordview.sys_language',
+            'config' => [
+                'type' => 'select',
+                'renderType' => 'selectSingle',
+                'foreign_table' => 'sys_language',
+                'items' => [
+                    ['LLL:EXT:tracking/Resources/Private/Language/locallang_tca.xlf:table.recordview.sys_language.0', 0],
+                ],
+                'readOnly' => true,
+            ]
+        ],
+        'user_agent' => [
+            'label' => 'LLL:EXT:tracking/Resources/Private/Language/locallang_tca.xlf:table.recordview.user_agent',
+            'config' => [
+                'type' => 'input',
+                'readOnly' => true,
+            ],
+        ],
+        'operating_system' => [
+            'label' => 'LLL:EXT:tracking/Resources/Private/Language/locallang_tca.xlf:table.recordview.operating_system',
+            'config' => [
+                'type' => 'input',
+                'readOnly' => true,
+            ],
+        ],
+        'url' => [
+            'label' => 'LLL:EXT:tracking/Resources/Private/Language/locallang_tca.xlf:table.recordview.url',
+            'config' => [
+                'readOnly' => true,
+                'type' => 'input',
+                'size' => 50,
+                'max' => 255,
```

Comment:
Isn't the url string limit a little short?

Of course we would likely prevent longer urls. But you can make such long urls in TYPO3.
Shouldn't this extension support that then?

Reply:
As we discussed the limit is only in TCA and not in SQL. So remove the `max` here and in the other pageview TCA configuration.